src/main/scala/net/kaliber/pdf/PdfRenderer.scala

```diff
@@ -1,23 +1,23 @@
 package net.kaliber.pdf
 
-import java.io.{
-  ByteArrayOutputStream,
-  StringReader,
-  StringWriter
-}
+import java.io.{ByteArrayOutputStream, StringReader, StringWriter}
+
 import org.w3c.tidy.Tidy
 import org.xhtmlrenderer.pdf.ITextRenderer
 import org.xhtmlrenderer.resource.XMLResource
 import org.xhtmlrenderer.context.StyleReference
+import org.xhtmlrenderer.extend.FontResolver
 
 /**
  * Simple wrapper.
  *
- * @classLoader The class loader used to resolve assets
+ * `PdfRenderer` wraps the the rendering with iText
```

Comment:
When I started to read the code I had put some comments for me to understand, I must've stayed on this mindset when I left the comment. :frowning: